ci/ray_ci/linux_container.py

```diff
@@ -28,6 +28,8 @@ def __init__(
         self.tmp_filesystem = tmp_filesystem
 
     def install_ray(self, build_type: Optional[str] = None) -> List[str]:
+        cache_readonly = os.environ.get("BUILDKITE_CACHE_READONLY", "")
```

Comment:
i think `BUILDKITE_CACHE_READONLY` need to be added to the env list to pass into the docker image somewhere; probably in container.py?, maybe search for `BUILDKITE_PIPELINE_ID`.

Reply:
this is used in the "build ray before testing" step, which runs in the forge, and `rayci` pipeline generator makes sure that builtin env var like this will pass through the docker plugin:

https://buildkite.com/ray-project/microcheck/builds/1632#019013fe-66e1-470a-95db-e0063b3128df/329-539